hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -892,6 +893,9 @@ message SnapshotInfo {
   // note: shared sizes can be calculated from: referenced - exclusive
   optional bool deepCleanedDeletedDir = 19;
   optional bytes lastTransactionInfo = 20;
+  optional uint64 exclusiveDirSize = 21;
```

Comment:
Thanks for the patch Swami. But I think I'm missing some important context here.

1. What is the definition of `exclusiveDirSize` ?
2. How does it differ from `exclusiveSize` ?